fuzzers/aflplusplus/builder.Dockerfile

```diff
@@ -15,18 +15,21 @@
 ARG parent_image=gcr.io/fuzzbench/base-builder
 FROM $parent_image
 
+# Install wget to download afl_driver.cpp. Install libstdc++ to use llvm_mode.
+RUN apt-get update && \
+    apt-get install wget libstdc++-5-dev -y
```

Comment:
Don't we have libstdc++ from OSS-Fuzz's clang image? This feels like we would need to do in every afl builder file, should this move to base-builder image ?

Reply:
No we have libc++.
I don't know why using libc++ doesn't work. I (like @kcc pointed out in our meeting) find it odd that the LLVM pass can't use LLVM's C++ standard library. I may look into this and send a patch as well.

Reply:
We don't need libstdc++ in every image because using libc++ works in most cases.